### ISSUE TITLE
apostrophe create-piece overwrites existing files if they already exist

### ISSUE THREAD

**Issue by bobclewell** [apostrophecms/apostrophe-cli#27]
When running `create-piece` the CLI just overwrites existing files without warning. It seems like a warning here would be a reasonable poke yoke, to prevent a user from potentially losing work.

I haven't checked to see if this is true about the other CLI `create` commands, but it probably is, and again a warning poke yoke on these would be helpful.

**Comment by boutell** [comment]
You're right, it should ask! I would say priority isn't super high because using git is standard practice, but the CLI is used most often by newcomers who might not have the best git hygiene, and even if you're a git pro it's annoying and you might not discover it until late in the game.

**Comment by stuartromanek** [comment]
this is unexpected and bad behavior even in the presence of git
